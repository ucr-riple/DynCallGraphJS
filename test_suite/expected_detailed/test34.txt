{
    'global (test34.js:1:1:12:3)': [
        'anon (test34.js:1:2:5:2)'
    ],
    'anon (test34.js:2:3:4:7)': [
        'anon (test34.js:2:4:4:4)'
    ],
    'anon (test34.js:3:5:3:13)': [
        'anon (test34.js:6:3:8:4)'
    ]
}