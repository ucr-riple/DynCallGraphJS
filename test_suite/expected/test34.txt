{ '(test34.js:1:1:12:3)':
   [ '(test34.js:1:2:5:2)' ],
  '(test34.js:2:3:4:7)':
   [ '(test34.js:2:4:4:4)' ],
  '(test34.js:3:5:3:13)':
   [ '(test34.js:6:3:8:4)' ] }